File: Arcade/Intro/17_arrayChange.js
function arrayChange(inputArray) {
    var moves = 0;
    for (var i = 1; i < inputArray.length; i++) {
        if (inputArray[i-1] >= inputArray[i]) {
            moves += ((inputArray[i-1] + 1) - inputArray[i]);
            inputArray[i] = inputArray[i-1] + 1;
        }
    } return moves;
}

File: Arcade/Intro/C++/17_arrayChange.cpp
int arrayChange(std::vector<int> inputArray) {
    int result=0;
    for(int i=1;i<inputArray.size();i++){
        int min=inputArray[i-1]+1;
        if(min>inputArray[i]){
        result+=min-inputArray[i];
        inputArray[i]=min;
        }
    }
    return result;
}
